### ISSUE TITLE
Investigation on observation normalization

### ISSUE THREAD

**Issue by zuoxingdong** [ikostrikov/pytorch-twin-sac#1]
First of all, thanks a lot for the great work. I got a quick question, recent days I've been trying out normalizing observation (subtracted by mean and divided by standard deviation), but it seems even degrades the performance. This trick helps a lot in policy gradient methods, so I'm wondering if DDPG can also benefit from it.

**Comment by ikostrikov** [comment]
One of the reasons it helps for PG methods is that in the beginning of training we sample from our initial policy. If the inputs are not normalized or it was poorly initialized, it may collapse.

For off-policy methods it is common to sample N initial actions from a random policy (for example, uniform distribution defined on the action space). Thus, the issue is not that severe as for on-policy.

In my experience, observation normalization helps for off-policy algorithm in certain cases but the different is often marginal.

**Comment by zuoxingdong** [comment]
Agreed, perhaps for the environments where the observations have different magnitude across different dimensions (NN often suffer from fitting such input data), it might be sufficient to scale the magnitude to [-1, 1], e.g. Mujoco tasks.
